files/build_templates/docker_image_ctl.j2

```diff
@@ -302,6 +302,10 @@ start() {
     if [ ! -f /etc/sonic/zero_profiles.json ] && [ -f /usr/share/sonic/templates/zero_profiles.j2 ]; then
         sonic-cfggen -d -t /usr/share/sonic/device/$PLATFORM/zero_profiles.j2 > /etc/sonic/zero_profiles.json
     fi
+
+    {%- if enable_asan == "y" %}
+    mkdir -p /var/log/asan
```

Comment:
is there way to have this as a constant/define to be shared with swss on another PR you created?

Reply:
I changed the way how to pass address sanitizer options to applications. Now they come via the environment variable defined in this repo (in supervisord.conf file).